### PR TITLE
fix(docs): update PR template to make it simpler

### PR DESCRIPTION
**Pull request by lahabana**
Remove standard body and replace by a checklist as an array

Signed-off-by: Charly Molter <charly.molter@konghq.com>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
+| 2. Link to UI issue or PR if needed                                                                   |        |          |                                          |
+| 3. Is the [issue worked on linked][1]                                                                 |        |          |                                          |
+| 4. Unit Tests                                                                                         |        |          |                                          |
+| 5. E2E Tests                                                                                          |        |          |                                          |
+| 6. Manual Universal Tests                                                                             |        |          |                                          |
+| 7. Manual Kubernetes Tests                                                                            |        |          |                                          |
+| 8. Do you need to update [`UPGRADE.md](/UPGRADE.md)?                                                  |        |          |                                          |
```
Comment (slonka) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: nitpick: extra `

Comment (slonka) [reply .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
| 8. Do you need to update [UPGRADE.md](/UPGRADE.md)?                                                  |        |          |                                          |
```

**Review by slonka — APPROVED** [review]
I think I sometimes used checkboxes to fill this out but can be Y/N/NA. If others feel strongly about checkboxes let's go back to checkboxes otherwise 👍

**Review by jakubdyszkiewicz — COMMENTED** [review]
So as a reviewer, should I fill the reviewer column?
I think the reviewer should double-check what was filled by the author, no?

**Comment by michaelbeaumont** [comment]
Not sure about having a reviewer column, especially since there can be multiple reviewers.

A reviewer checklist is probably something that works better with a github action that automatically replies to reviews with a checklist or with [GitHub saved replies](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/using-saved-replies)

**Comment by lahabana** [comment]
So should I drop the reviewer column? I'm fine with it. How do you feel about Y/N/NA I find NA is useful as it's often the case

**Comment by lahabana** [comment]
@michaelbeaumont @jakubdyszkiewicz @slonka WDYT now?

**Review by michaelbeaumont — APPROVED** [review]

**Comment by slonka** [comment]
Checkboxes better, no reviewer column - also ok with me so 👍

**Pull request merged**